src/util.cpp

```diff
@@ -4801,6 +4802,12 @@ bool resolveLink(/* in */ const char *scName,
     resAnchor=cd->anchor();
     return TRUE;
   }
+  else if ((cd=getClass(linkRefPHP)) && (SrcLangExt_PHP==cd->getLanguage())) // class link
```

Comment:
```suggestion
  else if ((cd=getClass(linkRefPHP)) && (SrcLangExt::PHP==cd->getLanguage())) // class link
```
This has changed in the meantime.